scraper/dev/docker/Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM algolia/base-documentation-scrapper
+FROM algolia/base-documentation-scraper
 MAINTAINER Algolia <documentationsearch@algolia.com>
-ARG USE_PYTHON3=false
 
-# Install correct python version and dependencies
-RUN sh /root/setup_python.sh $USE_PYTHON3
+WORKDIR /root
```

Comment:
:+1: Much better than adding `/root/` in all following commands. Well done.

Reply:
thanks